src/test/java/edu/unc/lib/boxc/migration/cdm/CdmExportCommandIT.java

```diff
@@ -51,6 +51,8 @@
 
     private final static String COLLECTION_ID = "my_coll";
     private final static String PASSWORD = "supersecret";
+    public static final String CDM_QUERY_BASE = "/dmwebservices/index.php?q=dmQuery/"
```

Comment:
I think instead of making this a new constant, you can use the one you made in the service class. Like `CdmListIdService.CDM_QUERY_BASE` might be the right syntax?